dao/application_authentication_dao_test.go

```diff
@@ -250,6 +238,7 @@ func TestApplicationAuthenticationsByAuthenticationsDatabase(t *testing.T) {
 // and correct count of returned objects
 func TestApplicationAuthenticationListOffsetAndLimit(t *testing.T) {
 	testutils.SkipIfNotRunningIntegrationTests(t)
+	testutils.SkipIfNotSecretStoreDatabase(t)
```

Comment:
not sure why we need to skip this test when SECRET_STORE is not database?

Reply:
If you take a look at the `dao/application_authentication_dao#ApplicationAuthenticationsByAuthentications` function, you'll see that we've got a `query.Where("authentication_uid IN ?", authUuids)` condition when we're using Vault. That column does not exist yet, and therefore was giving me problems when running the tests.

Once we start using Vault, we will need to add a test for that one too.